pacman/pacman-canvas.js

```diff
@@ -178,7 +177,6 @@ function geronimo() {
 		$('#canvas-overlay-container').show();
 		$('#splash-screen').show();
 		$('#email-form').show();
```

Comment:
Do we need to remove this line?

Reply:
This is the name of the form for entering your player name, so no

Reply:
i see